### PR TITLE
Add FF Requset subscription and add multiple column lookup for attach…

### PR DESCRIPTION
**Pull request by bdjilka**
…ment lookup

### CODE REVIEW

**Review by zzzevaka — CHANGES_REQUESTED** [review]

File: connect_transformations/attachment_lookup/mixins.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def preload_attachment_for_lookup(self):
             self.excel_attachments_data = {}
 
             settings = self.transformation_request['transformation']['settings']
-            map_by = settings['map_by']['attachment_column']
+            map_by = [item['attachment_column'] for item in settings['map_by']]
```
[the remark this answers is not in the document]
Comment (zzzevaka) [reply connect_transformations/attachment_lookup/mixins.py]: fixed

**Review by zzzevaka — APPROVED** [review]

File: ui/src/pages/transformations/lookup_ff_request.js [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+        const option = document.createElement('option');
+        option.value = column.id;
+        option.text = getColumnLabel(column);
+        document.getElementById('parameter_column').appendChild(option);
+      });
+    }
+
+    if (settings) {
+      document.getElementById('criteria').value = settings.asset_type;
+      const columnId = columns.find((c) => c.name === settings.asset_column).id;
```
[the remark this answers is not in the document]
Comment (zzzevaka) [reply ui/src/pages/transformations/lookup_ff_request.js:322]: `settings.asset_column` can be null in the case of `Skip search by Subscription`.

**Review by zzzevaka — CHANGES_REQUESTED** [review]

File: ui/src/pages/transformations/lookup_ff_request.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (bdjilka) [reply ui/src/pages/transformations/lookup_ff_request.js:322]: As this was requested to unblock our colleagues, maybe worth to commit now as it is and then add fixes without a rush. WDYT?

[the remark this answers is not in the document]
Comment (zzzevaka) [reply ui/src/pages/transformations/lookup_ff_request.js]: [I have fixed](https://github.com/cloudblue/connect-transformations/pull/89/commits/9c85efca9283e828899c31ec054896b798564529)

**Pull request merged**